power_grid_model_c/power_grid_model/include/power_grid_model/component/transformer.hpp

```diff
@@ -258,6 +258,21 @@ class Transformer : public Branch {
             param0.ytt() = y0_series;
         }
 
+        auto const low_susceptance = -transformer_low_susceptance_ratio * sn_ / base_power_3p / uk_;
+        bool const zero_seq_available_from =
+            (winding_from_ == WindingType::wye_n && winding_to_ == WindingType::wye_n) ||
+            (winding_from_ == WindingType::wye_n && winding_to_ == WindingType::delta) ||
+            winding_from_ == WindingType::zigzag_n;
+        bool const zero_seq_available_to = (winding_to_ == WindingType::wye_n && winding_from_ == WindingType::wye_n) ||
+                                           (winding_to_ == WindingType::wye_n && winding_from_ == WindingType::delta) ||
+                                           winding_to_ == WindingType::zigzag_n;
```

Comment:
maybe more readable (optionally move `zero_seq_available` to separate function):

```suggestion
        auto const zero_seq_available = [](WindingType this_side, WindingType other_side) {
            using enum WindingType;
            switch (winding_from_) {
            case wye_n:
                return winding_to_ == wye_n || winding_to_ == delta;
            case zigzag_n:
                return true;
            default:
                return false;
            }
        }();
        bool const zero_seq_available_from = zero_seq_available(winding_from_, winding_to_);
        bool const zero_seq_available_to = zero_seq_available(winding_to_, winding_from_);
```